scripts/validate.py

```
@@ -161,24 +184,21 @@ def generate_manifest(self, output_path: Path = None):
         summary = results["summary"]
-        print(f"\n📊 Validation Summary:")
-        print(f"  Total files: {results['total_files']}")
-        print(f"  Valid: {summary['valid_files']} ✓")
-        print(f"  Invalid: {summary['invalid_files']} ✗")
-        print(f"  Warnings: {summary['total_warnings']}")
-        print(f"  Total size: {summary['total_size_bytes'] / 1024 / 1024:.2f} MB")
-        print(f"  Total entries: {summary['total_entries']:,}")
-        print(f"  Unique entries: {summary['total_unique_entries']:,}")
-
-        # Save manifest
+        print(f"\nValidation Summary:")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)